modules/hal_nordic/nrfx/nrfx_config.h

```diff
@@ -9,6 +9,12 @@
 
 #include <zephyr/devicetree.h>
 
+/*
+ * NRFX API version 2.9 flag.
+ * When the flag is set NRFX API is compatible with the previous NRFX release.
+ */
+#define NRFX_CONFIG_API_VER_2_9 1
```

Comment:
shouldn't we switch to nrfx 2.10 API immediately?

Reply:
No, zephyr and its modules are not aligned with API changes. When proper changes will be applied then `NRFX_CONFIG_API_VER_2_10` symbol should be set to 1